packages/backend-defaults/src/entrypoints/httpAuth/httpAuthServiceFactory.ts

```diff
@@ -71,23 +70,54 @@ type RequestWithCredentials = Request & {
   [limitedCredentialsSymbol]?: Promise<BackstageCredentials>;
 };
 
-class DefaultHttpAuthService implements HttpAuthService {
+/**
+ * @public
+ * Options for creating a DefaultHttpAuthService.
+ */
+export interface DefaultHttpAuthServiceOptions {
+  auth: AuthService;
+  discovery: DiscoveryService;
+  pluginId: string;
+  // Optionally override the default token extraction logic
+  extractTokenFromRequest?: (req: Request) => string | undefined;
```

Comment:
```suggestion
  getTokenFromRequestHandler?: (req: Request) => string | undefined;
```

```suggestion
  getTokenFromRequest?: (req: Request) => string | undefined;
```

```suggestion
  getToken?: (req: Request) => string | undefined;
```

Are any of these better?